### PR TITLE
Add Front-Commerce detection

### PR DESCRIPTION
**Pull request by paul-vd**
### Website
- https://front-commerce.com
- https://developers.front-commerce.com/

### Examples
- https://magento2.front-commerce.app/
- https://magento1.front-commerce.app/
- https://magento2-b2b.front-commerce.app/

### CODE REVIEW

File: src/technologies/f.json [hunk 1]
```diff
@@ -2349,5 +2349,24 @@
...
+    "cats": [
+      12,
+      18,
+      22,
+      57,
+      106,
+      108
+    ],
+    "description": "An API-first, headless e-commerce framework with PWA capabilities and a fast, customizable frontend.",
+    "icon": "front-commerce.svg",
```
Comment (nurbek91) [inline src/technologies/f.json:2363]: Front-Commerce
32x32

File: src/technologies/f.json [hunk 2]
```diff
@@ -2349,5 +2349,24 @@
...
+  },
+  "Front-Commerce": {
+    "cats": [
+      12,
+      18,
+      22,
+      57,
+      106,
+      108
+    ],
```
Comment (nurbek91) [inline src/technologies/f.json:2361]: Leave only category 6
```
"cats": [
      6,
    ],
```

File: src/technologies/f.json [hunk 3]
```diff
@@ -2349,5 +2349,24 @@
...
+    "description": "An API-first, headless e-commerce framework with PWA capabilities and a fast, customizable frontend.",
```
Comment (nurbek91) [inline src/technologies/f.json:2362]: Front-Commerce is a React-based ecommerce framework that provides a development environment and tools for building online stores, offering seamless integration with backend systems through GraphQL and supporting popular ecommerce platforms like Magento and Pimcore.

File: src/technologies/f.json [hunk 4]
```diff
@@ -2349,5 +2349,24 @@
...
+    "icon": "front-commerce.svg",
+    "implies": [
+      "React",
+      "Webpack",
+      "Node.js"
+    ],
```
Comment (nurbek91) [inline src/technologies/f.json:2368]:
```
 "implies": [
      "React",
      "GraphQL",
      "Webpack",
      "Node.js"
    ],
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!

File: src/technologies/f.json (hunk 2 above)
Comment (paul-vd) [reply src/technologies/f.json:2361]: Hmm, would it not also deserve **12**, and **18**? and also maybe **22**?

In itself it is not e-commerce at all, but instead proves the tools as framework for other developers to build an e-commerce site using front-commerce, simular to remix / nextjs / blitzjs, it handles everything related to bundling, tools, server, client, caching, etc...  so only category 6 seems incorrect?

Also it's not limited to e-commerce, it has full cms integrations as well, for example wordpress, prismic, contentful.

File: src/technologies/f.json (hunk 3 above)
Comment (paul-vd) [reply src/technologies/f.json:2357]: I'm checking with the team for the best description, thanks for the proposal, it's indeed better 😄 ✍🏻  I will amend my PR once I got feedback from them.

File: src/technologies/f.json (hunk 2 above)
Comment (nurbek91) [reply src/technologies/f.json]: Hey @paul-vd
My mistake should probably leave the 108 category Ecommerce frontends

File: src/technologies/f.json (hunk 3 above)
Comment (nurbek91) [reply src/technologies/f.json:2357]: Yes, any description will do. 
Starting like->  Front-Commerce is a ..... 
and need use UK spelling

File: src/technologies/f.json (hunk 2 above)
Comment (nurbek91) [reply src/technologies/f.json]: https://www.wappalyzer.com/technologies/ecommerce-frontends/ if these are direct competitors, Front-Commerce  technology should also be on this list

File: src/technologies/f.json (hunk 3 above)
Comment (paul-vd) [reply src/technologies/f.json]: It has been updated 😉

**Review by nurbek91 — APPROVED** [review]

File: src/technologies/f.json (hunk 2 above)
Comment (nurbek91) [reply src/technologies/f.json]: I did not understand the question. But everything looks right now

Comment (paul-vd) [reply src/technologies/f.json]: The tracking of different competitors on https://www.wappalyzer.com/technologies/ecommerce-frontends, is it based on the defined category of each service?  for example **108**

Comment (nurbek91) [reply src/technologies/f.json]: Ah yes mate!

**Comment by AliasIO** [comment]
Thanks!

**Comment by paul-vd** [comment]
@AliasIO @nurbek91 we recently made a change in our packaging to improve loading performance, so the latest version of our service no longer has the link tag, I added a new PR to fix this: https://github.com/wappalyzer/wappalyzer/pull/7791

**Pull request merged**